src/common/modal/RefreshModal.vue

```diff
@@ -249,15 +264,23 @@ const checkIsSubmoduleLinkValid = async () => {
 }
 
 const searchSubmodule = async () => {
+  if (!submoduleName.value) return
+  const isSubmoduleWithThisNameExist = submodules.value.some(
+    item => item === submoduleName.value,
+  )
+  if (isSubmoduleWithThisNameExist) {
+    isSuchSubmoduleExistError.value = true
+    return
+  }
```

Comment:
I think we can use destructuring, as in the example above

Reply:
1. It's not consistent
2. This is an unnecessary complication of logic
3. This code will be with error. When you decompose a Proxy object, you get the value of the variable, but if you change it, this field in the Proxy will not change. You should change value from Proxy by setter.